live/live-root/usr/lib/dracut/modules.d/99agama-cmdline/save-agama-conf.sh

```diff
@@ -22,4 +22,6 @@ if [ -e /run/agama/copy_network ]; then
     mkdir -p "$NEWROOT/etc/NetworkManager/system-connections/"
     cp /run/NetworkManager/system-connections/* "$NEWROOT/etc/NetworkManager/system-connections/"
   fi
+else
+  : >/run/agama/not_copy_network
```

Comment:
Your "Solution" mention creating "`/run/agama/copy_network` always", I guess you need to update your text.

Reply:
Yes, it was not updated. Done, and thanks for reporting.